### PR TITLE
feat(conformity): enable conformity approval services

### PR DESCRIPTION
**Pull request by AnuragNagpure**
## Description

db seeding data need to get enhanced.

## Why

Adding required db seeding data for further enhancement in table portal.agreements

## Issue

N/A Ref: CPLP-3580

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreements.json [hunk 1]
```diff
@@ -130,5 +130,18 @@
...
+    "date_created": "2023-12-01 00:00:00.000000 +00:00",
+    "date_last_changed": "2023-12-01 00:00:00.000000 +00:00",
+    "agreement_status_id": 1,
+    "name": "I confirm that the service I want to offer has successfully received a Catena-X certificate issued by an official Conformity Assessment Body (CAB). I acknowledge to upload the certificate.",
+    "issuer_company_id": "2dc4249f-b5ca-4d42-bef1-7a7a950a4f87",
+    "use_case_id": null,
+    "document_id": null,
+    "mandatory": true
   }
+
```
Comment (Phil91) [inline src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreements.json:146]: please remove the empty line

**Review by Phil91 — CHANGES_REQUESTED** [review]

**Comment by Phil91** [comment]
@jjeroch is it correct that the new agreement isn't linked to a document and use case?

File: src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreements.json [hunk 2]
```diff
@@ -130,5 +130,18 @@
...
+  },
+  {
+    "id": "311aac58-932b-4622-be8b-34337e48d70C",
+    "agreement_category_id": 4,
+    "date_created": "2023-12-01 00:00:00.000000 +00:00",
+    "date_last_changed": "2023-12-01 00:00:00.000000 +00:00",
+    "agreement_status_id": 1,
+    "name": "I confirm that the service I want to offer has successfully received a Catena-X certificate issued by an official Conformity Assessment Body (CAB). I acknowledge to upload the certificate.",
+    "issuer_company_id": "2dc4249f-b5ca-4d42-bef1-7a7a950a4f87",
+    "use_case_id": null,
```
Comment (jjeroch) [inline src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreements.json:142]: useCaseId to be set to '1aacde78-35ec-4df3-ba1e-f988cddcbbd9'

**Review by jjeroch — CHANGES_REQUESTED** [review]

**Comment by jjeroch** [comment]
> @jjeroch is it correct that the new agreement isn't linked to a document and use case?

@Phil91  thanks for sharing, we need to set the useCaseId, the rest is fine. Comment for Anurag added

File: src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreements.json (hunk 1 above)
Comment (AnuragNagpure) [reply src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreements.json]: removed

File: src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreements.json (hunk 2 above)
Comment (AnuragNagpure) [reply src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreements.json]: useCaseId set

**Review by Phil91 — APPROVED** [review]

**Pull request merged**